src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/AdminTranscriptionSearchServiceImpl.java

```diff
@@ -2,24 +2,35 @@
 
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.common.entity.TranscriptionEntity;
+import uk.gov.hmcts.darts.common.exception.DartsApiException;
+import uk.gov.hmcts.darts.common.repository.TranscriptionRepository;
+import uk.gov.hmcts.darts.transcriptions.mapper.TranscriptionResponseMapper;
+import uk.gov.hmcts.darts.transcriptions.model.GetTranscriptionDetailResponse;
 import uk.gov.hmcts.darts.transcriptions.model.TranscriptionSearchRequest;
 import uk.gov.hmcts.darts.transcriptions.model.TranscriptionSearchResponse;
 import uk.gov.hmcts.darts.transcriptions.model.TranscriptionSearchResult;
 import uk.gov.hmcts.darts.transcriptions.service.AdminTranscriptionSearchService;
 import uk.gov.hmcts.darts.transcriptions.service.TranscriptionSearchQuery;
 
+import java.time.OffsetDateTime;
 import java.util.ArrayList;
 import java.util.List;
 
 import static java.util.Collections.emptyList;
 import static org.springframework.util.CollectionUtils.isEmpty;
+import static uk.gov.hmcts.darts.transcriptions.exception.TranscriptionApiError.TRANSCRIPTION_NOT_FOUND;
 
 @Service
 @RequiredArgsConstructor
 public class AdminTranscriptionSearchServiceImpl implements AdminTranscriptionSearchService {
```

Comment:
```suggestion
public class AdminTranscriptionServiceImpl implements AdminTranscriptionSearchService {
```

Perhaps it would be better to name this service in a more generic way so that in the future we can add other type of admin transcriptions operations that are not a search?